packages/rest/tests/credential.test.ts

```diff
@@ -39,6 +39,48 @@ describe('CredentialController', () => {
     })
   })
 
+  describe('Get all credentials by state', () => {
+    test('should return all credentials', async () => {
```

Comment:
As we mock the results, we aren't testing whether te query is correctly applied here.

I think we should spy on the credential repository and see if it was called with the correct parameters:

```ts
  describe('Get all credentials by state', () => {
    test('should return all credentials', async () => {
      const credentialRepository = bobAgent.dependencyManager.resolve(CredentialRepository)
      const findByQuerySpy = jest.spyOn(credentialRepository, 'findByQuery').mockResolvedValueOnce([testCredential])

      const response = await request(app).get('/credentials').query({ state: testCredential.state })

      expect(findByQuerySpy).toBeCalledWith({
        state: testCredential.state,
      })

      expect(response.statusCode).toBe(200)
      expect(response.body).toEqual([testCredential].map(objectToJson))
    })
  })
```

Reply:
Got it! I'll make the changes now